yourgov/components/StaffMembersTable/SortBySelect.tsx

```diff
@@ -0,0 +1,66 @@
+import { Select } from '@ag.ds-next/react/src/select';
+import { StaffMember } from './lib/types';
+import { useSortAndFilterContext } from './lib/contexts';
+import { tableId } from './DataTable';
+
+export const SortBySelect = () => {
+	const { sort, setSort } = useSortAndFilterContext();
+
+	return (
+		<Select
+			label="Sort by"
+			hideOptionalLabel
+			options={[
+				{
+					value: 'name-ASC',
+					label: 'Name (A-Z)',
+				},
+				{
+					value: 'name-DESC',
+					label: 'Name (Z-A)',
+				},
+				{
+					value: 'role-ASC',
+					label: 'Role (A-Z)',
+				},
+				{
+					value: 'role-DESC',
+					label: 'Role (Z-A)',
+				},
+				{
+					value: 'status-ASC',
```

Comment:
Status is sortable in the select, but not in the table header 
<img width="625" alt="image" src="https://github.com/user-attachments/assets/5e577e87-0e27-4070-a8ee-d94e43162a33">

Reply:
- [x] Status th sortable